KalturaPlayerSDK.podspec

```diff
@@ -43,7 +43,7 @@ s.requires_arc = true
 # ――― Source Code ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 s.source_files  = "**/*.{h,m}", "PlayerSDK/KALTURAPlayerSDK/**/*.{h,m}"
 s.vendored_library = 'libWViPhoneAPI.a'
-s.resource_bundle = { 'KALTURAPlayerSDKResources' => 'KALTURAPlayerSDKResources/*.*' }
+s.resource_bundle = { 'KALTURAPlayerSDKResources' => 'KALTURAPlayerSDK/*.{xib,plist}' }
 #s.exclude_files = "Classes/Exclude"
```

Comment:
in that case, all h & m files in that directory will be copied to the resources target. You may want to move the plist and xib files to the KALTURAPlayerSDKResources directory (as original intended?) and then *.* will work